catalog-info.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: ember-sortable
+  description: |
+    Ember addon for sortable UI primitives.
+    This fork fixes many issues we experienced over the past years with ember-sortable.
+    The initial reason why we created this fork is a bit difficult to track down since we did not document it.
```

Comment:
Feels a bit weak. I'd remove this line since the line above it explains the purpose.